src/pages/utilities/config.md

```diff
@@ -0,0 +1,68 @@
+---
+previousText: ''
+previousUrl: ''
+nextText: ''
+nextUrl: ''
+---
+
+# Config
+
+The Config lets you configure your entire app. You can set the tab button layout, icon mode, animations, and more here.
+
+## Usage
+
+```typescript
+import { IonicModule } from '@ionic/angular';
+
+@NgModule({
+  ...
+  imports: [
+    BrowserModule, 
+    IonicModule.forRoot({
+      rippleEffect: false,
+      mode: 'md'
+    }), 
+    AppRoutingModule
+  ],
+  ...
+})
+```
+
+In this example, we are disabling the material design ripple effect across the app as well as forcing the app mode to be material design.
+
+## Config Options
+
+Below is a list of config options that Ionic uses.
+
+| Config                   | type               | Description                                                                                              |
+|--------------------------|--------------------|----------------------------------------------------------------------------------------------------------|
+| `actionSheetEnter`       | `AnimationBuilder` | Provides a custom enter animation for all `ion-action-sheet`, overriding the default "animation".
+| `actionSheetLeave`       | `AnimationBuilder` | Provides a custom leave animation for all `ion-action-sheet`, overriding the default "animation".
+| `alertEnter`             | `AnimationBuilder` | Provides a custom enter animation for all `ion-alert`, overriding the default "animation".
+| `alertLeave`             | `AnimationBuilder` | Provides a custom leave animation for all `ion-alert`, overriding the default "animation".
+| `animated`               | `boolean`          | When it's set to `false`, disables all animation and transition across the app.
+| `backButtonIcon`         | `string`           | Overrides the default icon in all `<ion-back-button>` components.
+| `backButtonText`         | `string`           | Overrides the default text in all `<ion-back-button>` components.
+| `hardwareBackButton`     | `boolean`          | Wherever ionic will respond to hardware go back buttons in an Android device.
```

Comment:
This description doesn't make sense. I think the booleans should be set up in `if` statements like:

```
If `true`, Ionic will respond to the hardware back button in an Android device
```